### PR TITLE
Caching improvements

### PR DESCRIPTION
**Pull request by mfournier**

### CODE REVIEW

File: common.ini.in [hunk 1]
```diff
@@ -43,6 +43,8 @@ redis.socket_connect_timeout = 0.1
 redis.socket_timeout = 0.25
 # timeout waiting for an available slot in the connection pool (seconds)
 redis.pool_timeout = 0.5
+# cache keys TTL (seconds). Used by redis' eviction mechanism, not by dogpile
+redis.expiration_time = 86400
```
Comment (tsauerwein) [inline common.ini.in]: So, you no longer want to use a LRU strategy? From the way we are using the cache, there would be no need to set an expire time. If a value changes, we change the cache key.

**Comment by tsauerwein** [comment]
Thanks for improving the timeout settings!

**Comment by mfournier** [comment]
On Mon, Dec 19, 2016 at 12:38:42AM -0800, Tobias Sauerwein wrote:
> 
> So, you no longer want to use a LRU strategy? From the way we are using
> the cache, there would be no need to set an expire time. If a value
> changes, we change the cache key.

I essentially want to have the flexibility to switch to a TTL-based
expiration strategy, if we finally don't find a good balance with the LRU
one.

I set the default cache TTL to one day, but this could also be one week.
Basically: give redis a hint about what data to remove first when it needs
to make room for new keys.

**Comment by tsauerwein** [comment]
Ok, thanks for the explanation. Feel free to merge.

**Pull request merged**